scripts/dev/build_and_deploy_operator.py

```diff
@@ -47,12 +47,12 @@ def _ensure_crds():
     crdv1 = client.ApiextensionsV1beta1Api()
     crd = _load_mongodb_crd()
 
-    ignore_if_doesnt_exist(
+    k8s_conditions.ignore_if_doesnt_exist(
         lambda: crdv1.delete_custom_resource_definition("mongodb.mongodb.com")
     )
 
     # Make sure that the CRD has being deleted before trying to create it again
-    if not wait_for_condition(
+    if not k8s_conditions.wait_for_condition(
```

Comment:
[nit] maybe wait_for_condition can be renamed just wait?

Reply:
or even `k8s_conditions .wait_for(cond)`, either way, works well 👍